source/egen-integrasjon/sikkerhet.rst

```diff
@@ -12,7 +12,7 @@ See also `approved organization certificate in the public sector <https://www.re
 
 Most HTTP clients have built-in support for two-way TLS. You can see examples of the implementation in `the client library written in C# <https://github.com/digipost/signature-api-client-dotnet/>`_ or `the client library written in Java <https://github.com/digipost/signature-api-client-java/>`_. The digital signature service only supports TLS 1.2 for two-way TLS.
 
-You use your own certificate in ``keystore`` (for verification of your identity), og legger til `the trust anchors (CA certificates) <http://begrep.difi.no/SikkerDigitalPost/1.2.0/sikkerhet/sertifikathandtering>`__ to the ``truststore`` (with which the server will verify its identity). Your certificate will be used for verification of your identity towards the server and the server will use Posten Norge AS' certificate to verify its identity. Since the trust anchors are in the ``truststore`` you will obtain most of the validation from there (provided that your language/framework handles this). What you have to do manually is to validate that the certificate belongs to Posten Norge AS by checking the organization number stated in the ``Common Name``.
+You use your own certificate in ``keystore`` (for verification of your identity), and add `the trust anchors (CA certificates) <http://begrep.difi.no/SikkerDigitalPost/1.2.0/sikkerhet/sertifikathandtering>`__ to the ``truststore`` (with which the server will verify its identity). Your certificate will be used for verification of your identity towards the server and the server will use Posten Bring AS' certificate to verify its identity. (Please note: Posten Bring AS was until June 6th 2023 called Posten Norge AS. During a transitional period, the name given on the certificate will remain Posten Norge AS.) Since the trust anchors are in the ``truststore`` you will obtain most of the validation from there (provided that your language/framework handles this). What you have to do manually is to validate that the certificate belongs to Posten Bring AS by checking the organization number stated in the ``Common Name``.
```

Comment:
Noe sånt?

```suggestion
You use your own certificate in ``keystore`` (for verification of your identity), and add `the trust anchors (CA certificates) <http://begrep.difi.no/SikkerDigitalPost/1.2.0/sikkerhet/sertifikathandtering>`__ to the ``truststore`` (with which the server will verify its identity). Your certificate will be used for verification of your identity towards the server and the server will use Posten Bring AS' certificate to verify its identity. (Please note: During a transitional period following the change of name to Posten Bring AS on June 6th 2023, the subject of the certificate will remain as "Posten Norge AS", though this name is not used for validating the certificate.) Since the trust anchors are in the ``truststore`` you will obtain most of the validation from there (provided that your language/framework handles this). What you have to do manually is to validate that the certificate belongs to Posten Bring AS by checking the organization number stated in the ``Common Name``.
```

Reply:
Det var straks mye bedre!